Makefile

```diff
@@ -975,7 +975,9 @@ else
 	cp bin/gt $(prefix)/bin
 	$(STRIP) $(prefix)/bin/gt
 endif
-	cp -r gtdata $(prefix)/bin
+	test -d $(prefix)/share/genometools \
+	  || mkdir -p $(prefix)/share/genometools
+	cp -r gtdata $(prefix)/share/genometools
```

Comment:
To be paranoid, I prefer…
```
cp -a gtdata $(prefix)/share/genometools/
```
which fails if the target directory does not exist, unlike `$(prefix)/share/genometools` without the trailing slash.